### ISSUE TITLE
Implementation of 3D Convex Splatting

### ISSUE THREAD

**Issue by rvandeghen** [nerfstudio-project/gsplat#519]
Hello,

Are you interested to have an implementation of 3D Convex Splatting ([paper](https://arxiv.org/abs/2411.14974), [project page](https://convexsplatting.github.io/)) to gsplat ?

Renaud

**Comment by rvandeghen** [comment]
@maturk we think it should not be that difficult, because most of the implementation was some “straightforward“ changes wrt the 3DGS implementation.

**Comment by rvandeghen** [comment]
@AlexMorgand Yes sure. You can send me an email to discuss about it if you want ;)
cc @heldjan

**Comment by rvandeghen** [comment]
@TarzanZhao Nice !
You can send me an email at r.vandeghen@uliege.be

**Comment by TarzanZhao** [comment]
@rvandeghen Sent!

**Comment by xiemeilong** [comment]
Has anyone looked into [Deformable Beta Splatting](https://arxiv.org/pdf/2501.18630)? Does it offer sufficient advantages over Convex Splatting ?
